docs/sources/operator/getting-started.md

```diff
@@ -1,162 +1,151 @@
 ---
 aliases:
 - /docs/agent/latest/operator/getting-started/
-title: Installing Grafana Agent Operator
-weight: 100
+title: Install Grafana Agent Operator without Helm
+weight: 110
 ---
 
-# Installing Grafana Agent Operator
+# Install Grafana Agent Operator without Helm
 
-In this guide you'll learn how to deploy the [Grafana Agent Operator]({{< relref "./_index.md" >}}) into your Kubernetes cluster. This guide does *not* use Helm. To learn how to deploy Agent Operator using the [grafana-agent-operator Helm chart](https://github.com/grafana/helm-charts/tree/main/charts/agent-operator), please see [Installing Grafana Agent Operator with Helm]({{< relref "./helm-getting-started.md" >}}).
```

Comment:
Same than the title. I would just mention that alternatively you can install the operator using helm